src/main/java/application/AbstractBaseController.java

```diff
@@ -253,18 +253,22 @@ protected EntityDetailListResponse getInlineDetail(MenuSession menuSession) {
         if (detailId == null) {
             return null;
         }
-
         Detail persistentDetail = session.getDetail(detailId);
+        TreeReference reference = getReference(session, entityDatum);
 
-        EvaluationContext ec = session.getEvaluationContext();
-
-        TreeReference ref = entityDatum.getEntityFromID(ec, stepToFrame.getValue());
-        if (ref == null) {
-            return null;
+        EvaluationContext ec;
+        if (inline) {
+            ec = session.getEvaluationContext();
+            return new EntityDetailListResponse(persistentDetail.getFlattenedDetails(),
```

Comment:
I'm not 100% sure what the context is here, but on the inline popup we want to display the tabs, not the flattened detail. The flattened detail was actually just for j2me where we couldn't display tabs.